src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card_notification_configure.html.twig

```diff
@@ -6,3 +6,14 @@
 {% block addon_description %}
     {{module.attributes.warning}}
 {% endblock %}
+
+{# Display database version #}
+{% block addon_version %}
+  {% if module.attributes.productType == "service" %} bbb
+    {{ 'Service by '|trans({}, 'Admin.Modules.Feature') }}
+    <b>{{ module.attributes.author }}</b>
+  {% else %}
+    {{ 'v%version% - by '|trans({ '%version%' : module.database.version }, 'Admin.Modules.Feature') }}
+    <b>{{ module.attributes.author }}</b>
+  {% endif %}
+{% endblock %}
```

Comment:
We have a duplicated code here. Did you try to make this file extend from `card_manage_installed.html.twig` to check if it still works ?